### ISSUE TITLE
Editor - Table Tool Bar still active after closing ROM

### ISSUE THREAD

**Issue by dschultzca** [RomRaider/RomRaider#147]
Seen in commit e861b44 and maybe since splitting table data from table GUI.

1. Open a ROM
2. Open a Table and toggle scalings in Table Tool bar
3. Close ROM
4. Table Tool bar is still active and selecting a different scale causes an exception.
```
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
	at com.romraider.maps.Table3D.setCurrentScale(Table3D.java:331)
	at com.romraider.maps.Table.setScaleByName(Table.java:746)
	at com.romraider.swing.TableToolBar.itemStateChanged(TableToolBar.java:774)
	at javax.swing.JComboBox.fireItemStateChanged(JComboBox.java:1205)
	at javax.swing.JComboBox.selectedItemChanged(JComboBox.java:1253)
	at javax.swing.JComboBox.contentsChanged(JComboBox.java:1309)

```

**Comment by dschultzca** [comment]
The fix works better, but there is still a problem.  Open a ROM, click on a table in the tree to open it, click the tree again to hide the table.  The Table tool bar stays active even when you close the ROM.

**Comment by dschultzca** [comment]
Fixed in commit 57c3d23d8245f74f07eba784a598d8662de5dc52
Thanks
